app/views/directives/edit-environment-from.html

```diff
@@ -78,4 +84,44 @@
         ng-click="$ctrl.onAddRow()">{{ $ctrl.addRowLink }}</a>
     </div>
   </div>
+
+  <overlay-panel class="add-config-to-application" show-panel="$ctrl.overlayPanelVisible" show-close="true" handle-close="$ctrl.closeOverlayPanel">
+    <div class="dialog-title">
+      <h3>Value Details</h3>
+    </div>
+    <div class="modal-body">
+      <h4>{{$ctrl.overlayPaneEntryDetails.metadata.name}}
+        <small class="muted">- {{$ctrl.overlayPaneEntryDetails.kind | humanizeKind : true}}</small></h4>
+
+      <div ng-if="!($ctrl.overlayPaneEntryDetails.data | hashSize)" class="empty-state-message text-center">
```

Comment:
s/hashSize/size

Reply:
@spadgett any contrasting opinion to wrapping the size/lodash filter with ```hashSize```? Or we/I could do one big catch with a massive search and replace... I seem to keep making this slip up